src/Node.hpp

```diff
@@ -30,6 +30,7 @@
 #include "util/nodeinfo/NodeInfoBase.hpp"
 #include "util/registry/registry_impl.hpp"
 #include "util/port/PortListPublisherBase.hpp"
+#include "util/transfer_metadata.hpp"
```

Comment:
It should be possible to drop this line if `fillMetaData` is moved to `SubscriptionBase`.